src/services/users/users.service.js

```diff
@@ -7,7 +7,97 @@ module.exports = (app) => {
     const options = { Model, id: 'username' };
 
     // Initialize our service with any options it requires.
-    app.use('/users', createService(options));
+    const users = createService(options);
```

Comment:
This boilerplate is required?

Reply:
I need to setup docs directly inside service, so yeah it's required.